oktaawscli/aws_auth.py

```diff
@@ -124,7 +145,7 @@ def write_sts_token(self, profile, access_key_id, secret_access_key, session_tok
 
         with open(self.creds_file, 'w+') as configfile:
             config.write(configfile)
-        self.logger.info("Temporary credentials written to profile: %s" % profile)
+        print("Temporary credentials written to profile: %s" % profile)
```

Comment:
Why go from a logger to print?

Reply:
Moved this statement from logger to print to give the user more info on where credentials were being saved even if they didn't use `--verbose`.

Reply:
Okay. Didn't realize that logging was disabled unless verbose was turned on.